DRAFT.md

```diff
@@ -216,15 +216,23 @@ Note: As the null state of `indirect` and `polymorphic` is not observable, and
 access to a moved-from object is erroneous, `std::optional` can be specialized
 by implementers to exchange pointers on move construction and assignment.
 
-### Noexcept
-
-TODO
-
-### Allocators as template arguments
-
-TODO
-
-## Differences between `indirect` and `polymorphic`
+### `noexcept` and narrow contracts
+
+There is a guideline for C++ library design that member functions with narrow
```

Comment:
This perhaps overstates the existence of C++ library design guidelines, but alternative wording:

C++ library design guidelines recommend that member functions with narrow 
contracts (runtime-preconditions) should not be marked `noexcept`.